tests/mbed/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ foreach(testcase ${alltests})
 	if(${found_selftest} GREATER 0 )
 		get_testcase_name(${testcase} name "../../3rdparty/mbedtls/mbedtls/programs/test/")
 		add_enclave_test(tests/mbed/${name} libmbedtest_host libmbedtest-${name}_enc)
+		set_tests_properties(tests/mbed/${name} PROPERTIES  ENVIRONMENT "PROJECT_SOURCE_DIR=${CMAKE_SOURCE_DIR}/")
```

Comment:
I am not sure this is the right way to go about it. It's close, but I think we should instead we should use `target_compile_definitions` for the target created by `add_enclave_test` to define this at compile time. Since the test properties may only be set through e.g. CTest (which could get confusing later if someone runs a test directly). If we were sure that it's set even when a test is run without CTest, then this would be fine. But still, a compilation definition seems safer.